### PR TITLE
fix(learn): Update page titles

### PR DESCRIPTION
**Pull request by dipikabh**
### Description

In the sidebar in the [Learn](https://developer.mozilla.org/en-US/docs/Learn) area, each drop-down's "first item" matches the drop-down header. This PR fixes a few discrepancies where the first item's page title is slightly different.

### Motivation

To follow the same style for page titles

### Additional details

A few instances where the above described style is followed

<kbd>learnsidebar-1</kbd>

<kbd>learnsidebar-2</kbd>

<kbd>learnsidebar-4
</kbd>

<kbd>learnsidebar-3</kbd>

### CODE REVIEW

File: files/en-us/learn/forms/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Web forms — Working with user data
+title: Core forms learning pathway
```
Comment (estelle) [inline files/en-us/learn/forms/index.md:2]:
```suggestion
title: Web forms learning pathway
```

**Review by estelle — CHANGES_REQUESTED** [review]
The web forms should read "web forms". You may want to change the second half of the title, but the first two words should be the original. The rest of the PR looks good.

Edit: or "HTML forms"

**Review by hamishwillee — APPROVED** [review]
Other than Estelle's comment https://github.com/mdn/content/pull/32643/files#r1520191823 LTGM

File: files/en-us/learn/forms/index.md (hunk 1 above)
Comment (dipikabh) [reply files/en-us/learn/forms/index.md]: Thanks! I agree with your suggestion to go with "web forms". What do you think about the following title to align with other similar headings like "JavaScript building blocks" and "CSS building blocks": 

```suggestion
title: Web form building blocks
```
In my previous edit, I was trying to match the page title with the drop-down header, but ideally the header should be updated:

<kbd>learnsidebar-5
</kbd>

**Comment by dipikabh** [comment]
Thank you @estelle and @hamishwillee for the review.

**Review by estelle — APPROVED** [review]

**Comment by dipikabh** [comment]
Thanks, @estelle!

**Pull request merged**